Python/Product/PythonTools/PythonTools/Repl/PythonInteractiveEvaluator.cs

```diff
@@ -134,7 +134,17 @@ partial class PythonInteractiveEvaluator :
                 if (config == null) {
                     _analyzer = _serviceProvider.GetPythonToolsService().DefaultAnalyzer;
                 } else {
-                    _analyzer = new VsProjectAnalyzer(_serviceProvider, interpreterService.FindInterpreter(config.Interpreter.Id));
+                    Build.Evaluation.Project projectFile = null;
+                    var moniker = ProjectMoniker ??
```

Comment:
Can everything in here up to GetProjectFromFile(moniker) be factored into a method so it can be used both here and down below?